ast/misc.go

```diff
@@ -1981,18 +1981,53 @@ func (n *AdminStmt) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+// RoleOrPriv is a temporary structure to be further processed into auth.RoleIdentity or PrivElem
+type RoleOrPriv struct {
+	Symbols []string    // hold undecided symbols
```

Comment:
can this be a single string? mysql dynamic privilege is always a single identifier, and RDS's `LOAD FROM S3` / `SELECT INTO S3` / `INVOKE stuff` can be concatenated into a single string.